packages/java/parser-jvm-core/src/main/java/dev/hilla/parser/core/Parser.java

```diff
@@ -27,6 +31,22 @@ public final class Parser {
     private static final Logger logger = LoggerFactory.getLogger(Parser.class);
     private final Config config;
 
+    private static final String ENDPOINT_EXPOSED_AND_ACL_ANNOTATIONS_ERROR_TEMPLATE = """
+            Class `%s` is annotated with `%s` and `%s` annotation. %n
+            Classes annotated with `%s` should not contain any of access control annotations and %n
+            this exception is for preventing the confusion. The class level access control rules of the child class %n
```

Comment:
Nit: "should not" -> "must not" would be more appropriate given that doing so doesn't work at all. 

It's also not clear what is meant by "confusion", I think it would be OK to leave the reasoning out and just state the fact.

Reply:
Would something like the following be better?
```
... and this exception is for preventing the application startup with misconfiguration.
```

Reply:
Looks good 👍

Reply:
Done.